plantbuild/build.jsonnet

```diff
@@ -0,0 +1,8 @@
+local c = import 'dc.jsonnet';
+local dc = c {
+  dockerRegistry: '562055475000.dkr.ecr.ap-northeast-1.amazonaws.com',
```

Comment:
We need to move to a public registry later.

Reply:
https://theplanttokyo.atlassian.net/browse/SRE-2830